kiwix-build_ci/jammy_builder.dockerfile

```diff
@@ -32,8 +34,9 @@ RUN apt update -q \
   && rm -rf /var/lib/apt/lists/* /usr/share/doc/* /var/cache/debconf/* \
   && pip3 install meson pytest gcovr requests distro
 
-# QT_SELECT has to be set to Qt5 so Kiwix Linux/Windows compiles
-ENV QT_SELECT qt5
+# Set Qt6 per default (QT_SELECT has to be set to Qt5 so Kiwix Linux/Windows compiles)
+RUN qtchooser -install qt6 $(which qmake6)
+ENV QT_SELECT qt6
```

Comment:
Is the intent to make Qt6 the default with jammy, but then override the kiwix-desktop build on jammy to use qt5? Or to build with both Qt5 and Qt6 on jammy? Right now using the jammy container implies Qt5.

Reply:
The intent on Jammy is to:
* be able to build with either with qt6 or qt5
* per drfault, without any additional work it should comipile with qt6